CONTRIBUTING.rst

```diff
@@ -116,6 +122,23 @@ Finally, generate a new private key for the app, replace newlines with ``\n``,
 and set it as the ``DOCKER_GITHUB_APP_KEY`` environment variable (the entire
 key, not a path to one).
 
+Full Access App
+~~~~~~~~~~~~~~~
+
+The second GitHub app is used to create GitHub repositories in your organization
+when Projects are created in Metecho. The app will need the following
+permissions:
+
+- Repository permissions
+    - Administration: Read & write
+- Organization permissions
+    - Members: Read & write
```

Comment:
After writing this I wonder if it's even worth having two apps. We could add these permissions to the regular app, and determine if repository creation is supported based on the presence of these permissions 🤷

Reply:
Yeah, that makes some sense, if it's easy enough for us to quickly check permission support. (That reminds me -- if permissions are missing, I suppose we should hide the "Create Project" button entirely...)

Maybe something to run by SFDO on Wednesday?

Reply:
Added this to the agenda

Reply:
SFDO confirmed we should only use one app